src/test/java/com/jcabi/email/wire/SMTPTest.java

```diff
@@ -58,20 +58,34 @@
  * @since 1.0
  */
 public final class SMTPTest {
+    /**
+     * Bind address for the smtp server.
+     */
+    private static final String BIND_ADDRESS = "localhost";
+    /**
+     * Constant three.
+     */
+    private static final int THREE = 3;
```

Comment:
@amihaiemil please read http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#redundant-constants